test-app/package.json

```diff
@@ -57,7 +61,7 @@
     "ember-page-title": "8.0.0",
     "ember-qunit": "8.0.1",
     "ember-resolver": "11.0.1",
-    "ember-source": "4.12.0",
+    "ember-source": "5.3.0",
```

Comment:
I wonder if we wouldn't prefer bumping `ember-source` in a separate PR? wdyt?

Reply:
Especially because it also requires adding 2 new ember-try scenarios to the CI (for both 4.8 and 4.12 LTS versions)